admin/templates/common_auth/emails/invitation_email.html

```diff
@@ -0,0 +1 @@
+{% extends 'registration/password_reset_email.html' %}
```

Comment:
Sorry for the confusion - I believe I was going to extend the template further but the base customized one is now under admin/templates/emails/password_reset_email.html